### DISCUSSION TITLE
Kunstmaan Demo available?  Or Slack?

### DISCUSSION THREAD

**Discussion by ScottCrumpler** [Kunstmaan/KunstmaanBundlesCMS#3425] [Q&A] [answered]
I've only just recently discovered Kunstmaan CMS.  I've noticed that the public demo on the Kunstmaan site isn't working and the link to the Slack seems to be broken as well.  Are either of these still available?  I've been an avid user of another Symfony-based CMS whose future is now in question, so I'm really hoping to find another great Symfony-based CMS to fall in love with. :)

**Comment by ScottCrumpler** [comment]
bump

**Comment by mvaljento** [comment]
Is the project dead? I'm looking for a Symfony based CMS to integrate into an existing project, but can't find any suitable options.

**Comment by acrobat** [comment, marked as answer]
The demo itself is not hosted anymore as the hosting is not provided anymore. If you want to evaluate the CMS itself you can always install it locally and choose to install it with demo data. This produces the same website as previously used for the demo site.

See https://kunstmaanbundlescms.readthedocs.io/en/stable/installation/

Regarding slack, I think the invite url is expired again. I will update the website later, but for new here is a new working link https://join.slack.com/t/kunstmaancms/shared_invite/zt-2yxzf4t7q-TX_YYQ6UHASssd_z933ZrA
